opencog/nlp/scm/processing-utils.scm

```diff
@@ -21,26 +21,20 @@
    )
 )
 
-; -----------------------------------------------------------------------
-; global vars:
-; new-sent anchor points at the node to which all new sentences are connected
-;
-(define *new-parsed-sent-anchor* (AnchorNode "# New Parsed Sentence" (stv 1 1)))
```

Comment:
There are two reasons to not remove this:
1) if someone changes the text "# New Parsed Sentence" in one place, they have to remember to do a global search-n-replace everywhere.   So, this is the same reason one avoids literals in C++ code.
2) there is some very minor performance boost from not having to evaluate this expression every time it is used.